### PR TITLE
Update Japanese translation

### PR DESCRIPTION
**Pull request by taxi13245**
**Changes proposed in this pull request:**

-Fixed a syntax error in the Json file.
-Added missing Japanese translation.
-Corrected some spelling inconsistency.

**Ping @CosmoMyzrailGorynych**

### CODE REVIEW

File: app/data/i18n/Japanese.json [hunk 1]
```diff
@@ -556,6 +553,17 @@
...
+        "noShuttlesYet": "ビジネスシャトルはまだありません( ;∀;) 御社が第一号になるかもしれません",
+        "shuttlesDescription": "ビジネスシャトルは、ct.jsのパートナーとして考えられています。ct.jsのホームページや店舗ページで紹介されます。",
+        "spacePirates": "スペースパイレーツ",
+        "noPiratesYet": "スペースパイレーツはまだいません。",
+        "piratesDescription": "スペースパイレーツはDiscordサーバーで優先的にサポートされます。また、ここにリストアップされます。",
+        "spaceProgrammers": "スペースプログラマー",
+        "programmersDescription": "\"スペースプログラマー\" は、ct.js がオープンソース化され、パトロンにジャムゲームのソースがもたらされる前に利用可能だったレガシー層です。",
+        "aspiringAstronauts": "宇宙飛行士を目指す人",
+        "noAstronautsYet": "まだ、宇宙飛行士を目指す人はいません。",
+        "astronautsDescription": "宇宙飛行士はDiscordで特別な役割を与えられ、ここにリストアップされます。",
```
Comment (CosmoMyzrailGorynych) [inline app/data/i18n/Japanese.json]: These are outdated and must not be in new releases. Having these keys will break the build.

**Review by CosmoMyzrailGorynych — CHANGES_REQUESTED** [review]
```

Translation file Japanese.json
------------------------------
Coverage: 95% 💙  Fabulous!
Not translated: 26
  colorPicker.altClick
  exporterError
  intro.sponsoredBy
  intro.supportedBy
  languageSelector
  patreon.sponsors
  patreon.noSponsorsYet
  patreon.sponsorsDescription
  patreon.businessCats
  patreon.businessCatsDescription
  patreon.cats
  patreon.catsDescription
  mainMenu.settings.altTemplateLayout
  mainMenu.project.startNewWindow
  mainMenu.project.convertToJs
  mainMenu.project.confirmationConvertToJs
  roomTiles.addTileLayer
  roomTiles.addTileLayerFirst
  roomView.viewportHeading
  roomView.followTemplate
  roomView.followCodeHint
  textureView.slicing
  textureView.viewSettings
  textureView.exportSettings
  textureView.axisExplanation
  scriptables.jumpToProblem
Excess: 11 ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ 
  patreon.businessShuttles
  patreon.noShuttlesYet
  patreon.shuttlesDescription
  patreon.spacePirates
  patreon.noPiratesYet
  patreon.piratesDescription
  patreon.spaceProgrammers
  patreon.programmersDescription
  patreon.aspiringAstronauts
  patreon.noAstronautsYet
  patreon.astronautsDescription
```

File: app/data/i18n/Japanese.json (hunk 1 above)
Comment (taxi13245) [reply app/data/i18n/Japanese.json]: Thanks for the reply. How do I send the changes, do I just recreate the PullRequest again?

Comment (taxi13245) [reply app/data/i18n/Japanese.json]: Thanks for the reply.
I have added a new commit.
Please check it out.

**Comment by CosmoMyzrailGorynych** [comment]
I'll review it once I'm in an adequate work environment; I'm on a trip far from home rn. Sorry for the delays!

**Review by CosmoMyzrailGorynych — APPROVED** [review]
Looks good!

**Pull request merged**